# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/PBrowser.ipdl
# Commit: 791382b2c9bf
# Full Hash: 791382b2c9bf60371b68c0dd4d2ed26e4b846901
# Author: Barret Rennie <barret@brennie.ca>
# Date: 2019-04-04 04:33:36
# Regressor Bug: 1510569
# File Overlap Count: 1
# Description:
#   Bug 1510569 - Port onStatusChange notifications inside WebProgressChild.jsm to C++ r=baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D25446
# ==============================================================================

diff -r 31b206e2046f -r 791382b2c9bf dom/ipc/PBrowser.ipdl
--- a/dom/ipc/PBrowser.ipdl	Wed Apr 03 17:31:41 2019 +0000
+++ b/dom/ipc/PBrowser.ipdl	Wed Apr 03 17:31:54 2019 +0000
@@ -537,6 +537,10 @@
     async RegisterProtocolHandler(nsString scheme, nsIURI handlerURI, nsString title,
                                   nsIURI documentURI);
 
+    async OnStatusChange(WebProgressData? aWebProgressData,
+                         RequestData aRequestData, nsresult aStatus,
+                         nsString aMessage);
+
     async OnContentBlockingEvent(WebProgressData? aWebProgressData,
                                  RequestData aRequestData, uint32_t aEvent);
 